Server/VikingVault.DataAccess/Migrations/20190814073151_SeedAdmin.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VikingVault.DataAccess.Migrations
+{
+    public partial class SeedAdmin : Migration
```

Comment:
This migration does nothing. Please remove it.

Reply:
Done 👍